BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.java

```diff
@@ -331,4 +360,21 @@ Context getApplicationContext() {
     public void invalidateClientToken() {
         authorizationLoader.invalidateClientToken();
     }
+
+    boolean useDefaultDeepLinkHandler() {
+        return useDefaultDeepLinkHandler;
+    }
+
+    /**
+     * Set this property to true to enable default deep link handling for browser switched flows.
+     * For web flows, this means launching the browser in a task separate from the calling activity.
+     *
+     * NOTE: When default deep linking is enabled, all custom url schemes set in {@link BraintreeClient}
+     * constructors will be ignored.
+     *
+     * @param useDefaultDeepLinkHandler set to true to enable the default deep link handler. This value is false by default.
+     */
+    public void useDefaultDeepLinkHandler(boolean useDefaultDeepLinkHandler) {
```

Comment:
Calling the behavior from V3 the `default` behavior is confusing to me. Especially since merchants will have to opt-in to enable the "default" behavior.

Would it be more clear to call this something like `launchBrowsersAsNewTask(true)`?

Reply:
Yeah I'm up for renaming this. "Default Deep Link Handler" is the sentiment, but it is confusing – if this is set to `false` there is actual _default_ SDK behavior that already exists.